server/conf/application.conf

```diff
@@ -308,6 +308,28 @@ play.filters {
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
 
+  # GZip filter configuration
+  gzip {
+
+    # The maximum amount of data to send to the Gzip compressor in one go. Use `0` or `2147483647` to disable the buffering.
+    #
+    # In general, it is recommended to turn off the buffer and prevent generation of overlong chunks at the source.
+    bufferSize = 8k
+
+    # The maximum amount of content to buffer for gzipping in order to calculate the content length before falling back
+    # to chunked encoding.
+    chunkedThreshold = 100k
+
+    # The compression level to use, integer, -1 to 9, inclusive. See java.util.zip.Deflater.
+    compressionLevel = -1
+
+    # The byte threshold for the response body size which controls if a response should be gzipped (e.g. 1k).
+    # If the body size cannot be determined, then it is assumed the response is over the threshold.
+    # Set to 0 if you want to compress all responses, no matter how large the response body size is.
+    threshold = 0
```

Comment:
Set this to a low number (500 or 1000 is probably good).  There's no need to spend the extra effort compressing small files that don't get much of a size gain.

```
    threshold = 1000     # Don't compress very small responses.
```